src/linkingRe.re

```diff
@@ -0,0 +1,17 @@
+external openURL : string => Js.Promise.t unit =
+  "openUrl" [@@bs.scope "Linking"] [@@bs.module "react-native"];
+
+external _canOpenUrl : string => Js.Promise.t Js.boolean =
+  "canOpenUrl" [@@bs.scope "Linking"] [@@bs.module "react-native"];
+
+let canOpenUrl url =>
+  _canOpenUrl url |> Js.Promise.then_ (fun bool => Js.Promise.resolve (Js.to_bool bool));
```

Comment:
Nothing important but I used to do this, too until I noticed that you can simply create another function with the same name. It has the benefit that _canOpenUrl is shadowed when generating `rei`